src/couch/src/couch_file.erl

```diff
@@ -119,6 +121,7 @@ append_term_md5(Fd, Term, Options) ->
     Comp = couch_util:get_value(compression, Options, ?DEFAULT_COMPRESSION),
     append_binary_md5(Fd, couch_compress:compress(Term, Comp)).
 
+
```

Comment:
Extra whitespace? Wonder if you mean to move `append_terms` here at first so they go right below `append_term`

Reply:
I didn't do a good job cleaning up this PR since I don't expect it to be merged as is (because PSE). I had things in slightly different places as you said before moving them. The organization of couch_file's API isn't very clean so I had a few different spots for it before settling on where it is.

Reply:
Ah no worries. That makes sense, I should have paid more attention to the PR message